src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,35 @@
+.contacts {
```

Comment:
The block name should be more descriptive to make it easier to understand its purpose. Consider renaming it to 'eco-cosmetics-contacts' or similar.